src/main/scala/nl.knaw.dans.easy.bagstore/server/StoresServletComponent.scala

```diff
@@ -147,21 +147,28 @@ trait StoresServletComponent extends DebugEnhancedLogging {
       trace(())
       basicAuth()
       debug("Authenticated")
-      val bagstore = params("bagstore")
+      val bagStore = params("bagstore")
       val uuidStr = params("uuid")
-      logger.info(s"Received PUT request for bagstore $bagstore and UUID $uuidStr")
-      bagStores.getBaseDirByShortname(bagstore)
+      val contentType = Option(request.getHeader("Content-Type"))
+      logger.info(s"Received PUT request for bagstore $bagStore and UUID $uuidStr")
```

Comment:
@jo-pol is this logging still necessary? I thought we had servlet logging here, right?

Reply:
It looks like you are right, we don't need it any more.
![image](https://user-images.githubusercontent.com/10553630/52114521-8b4c3080-260c-11e9-9a0e-8560715bc073.png)

Reply:
Than ill will remove the log line